File: ASP.NET/test/Core/WebAPI/VSC/TodoApi/wwwroot/app/model/Personnel.js
Ext.define('TodoApi.model.Personnel', {
    extend: 'Ext.data.Model',

    schema: {
        namespace: 'TodoApi.model'
    },

    idProperty: "id",

    fields: [
            { name: "id", type: "int" },
            "name",
            "emai",
            "phone"
    ]
});

File: ASP.NET/test/Core/WebAPI/VSC/TodoApi/Models/Personnel.cs
namespace TodoApi
{
    public class Personnel
    {
        public int id {get; set;}
        public string name {get; set;}
        public string email{get; set;}
        public string phone{get; set;}

        public Personnel(int id = 0, string name = "", string email = "", string phone = "")
        {
            this.id = id;
            this.name = name;
            this.email = email;
            this.phone = phone;
        }
    }
}